src/components/ParaCard.vue

```
@@ -1,22 +1,38 @@
+  result: '',
   error: null,
 });
+
+const logger = createLogger('ParaCard');
+const isTab = ref('translation');
+
+const parsedResult = computed<ResponseFormatType>(() => {
+  try {
+    return JSON.parse(props.result);
+  } catch (error) {
+    logger.error`failed to parse result: ${error}`;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)